### DISCUSSION TITLE
Повідомлення форуму: class file ugQ_hgxb not exist on ./extensions/ugq_hgxb.php

### DISCUSSION THREAD

**Discussion by marchelloUA** [replace-org-ua/invites#14] [General]
Повідомлення форуму: class file ugQ_hgxb not exist on ./extensions/ugq_hgxb.php

**Comment by leofun01** [comment]
[Вже було](https://github.com/orgs/replace-org-ua/teams/members/discussions/9#discussion-9-comment-6).

**Comment by osbre** [comment]
Знову те саме. Хтось має інформацію про цю вразливість?

**Reply by leofun01** [reply]
```
class file sJ_pIxbt not exist on ./extensions/sj_pixbt.php
```
Завтра буду писати до `shanand6@gmail.com`.

**Reply by marchelloUA** [reply]
`Повідомлення форуму:
class file FI_UQMWX not exist on ./extensions/fi_uqmwx.php`

@leofun01 FYI

**Comment by marchelloUA** [comment]
Повідомлення форуму
class file K_lwrfo not exist on ./extensions/k_lwrfo.php

**Comment by marchelloUA** [comment]
Повідомлення форуму
class file c_PvN not exist on ./extensions/c_pvn.php

**Comment by marchelloUA** [comment]
Повідомлення форуму
class file aiD_JrSQ not exist on ./extensions/aid_jrsq.php

**Comment by marchelloUA** [comment]
Прохання розглянути можливість сконтактуватися з підтримкою хостера через власницю і попрохати створити розклад запуску антивіруса щодня.

**Comment by marchelloUA** [comment]
This site can’t be reachedThe webpage at https://replace.org.ua/search/recent/ might be temporarily down or it may have moved permanently to a new web address.
ERR_SOCKS_CONNECTION_FAILED
